api/utilities/response_generation.py

```
@@ -0,0 +1,445 @@
+        return {
+            "text": clarification_text,
+            "status": "needs_clarification",
+            "quality_assessment": quality_assessment,
+            "clarification_count": clarification_counter[user_id]
+        }
+
+    # If response is good quality, reset clarification counter and return response
+    clarification_counter[user_id] = 0
+
+    # Post-process the response (remove artifacts, fix formatting)
+    processed_response = post_process_response(raw_response, business_name)
```

Ruff (0.11.9)

Local variable `processed_response` is assigned to but never used

Remove assignment to unused variable `processed_response`

(F841)